plugins/modules/sonic_mclag.py

```diff
@@ -75,6 +75,10 @@
         description:
           - MCLAG session timeout value in secs.
         type: int
+      session_vrf:
+        description:
+        - MCLAG session vrf
+        type: str
```

Comment:
Sorry for not noticing this earlier:

Please add  a "version_added" attribute under the "session_vrf" attribute (at the same indentation as "description" and "type". This is needed for new attributes (options) added to a module after its initial release. (See "https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-fields".)

The version in which the "session_vrf" attribute will be released is 2.5.0, so please add the new attribute as shown below:

version_added: '2.5.0'

Reply:
Also:  Please re-run the regression test using the maximum verbosity option (-vvvv) and re-post the resulting regression summary (.html) file printed into ".pdf" format to provide the "Input", "Commands", "Before", and "After" dictionaries for each test section in the summary result output.

Reply:
I observe below error while installing ansible collections,
thenmozhi.gopal@sncinfra-maa-01:dellemc.enterprise_sonic$ ansible-galaxy collection install -vvv /neteng/thenmozhi.gopal/ansible_dir/dellemc.enterprise_sonic --force
[DEPRECATION WARNING]: Ansible will require Python 3.8 or newer on the controller starting with Ansible 2.12. Current version: 3.6.15 (default, Sep 23 2021, 15:41:43)
[GCC]. This feature will be removed from ansible-core in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
ansible-galaxy [core 2.11.12]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/thenmozhi.gopal/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible
  ansible collection location = /home/thenmozhi.gopal/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/thenmozhi.gopal/.local/bin/ansible-galaxy
  python version = 3.6.15 (default, Sep 23 2021, 15:41:43) [GCC]
  jinja version = 3.0.3
  libyaml = True
Using /etc/ansible/ansible.cfg as config file
Starting galaxy collection install process
Found installed collection ansible.utils:2.10.3 at '/home/thenmozhi.gopal/.ansible/collections/ansible_collections/ansible/utils'
Found installed collection ansible.netcommon:5.1.2 at '/home/thenmozhi.gopal/.ansible/collections/ansible_collections/ansible/netcommon'
Found installed collection dellemc.enterprise_sonic:2.2.0 at '/home/thenmozhi.gopal/.ansible/collections/ansible_collections/dellemc/enterprise_sonic'
Process install dependency map
Opened /home/thenmozhi.gopal/.ansible/galaxy_token
[WARNING]: Skipping Galaxy server https://galaxy.ansible.com/api/. Got an unexpected error when getting available versions of collection ansible.netcommon:
'/api/v3/plugin/ansible/content/published/collections/index/ansible/netcommon/versions/'
ERROR! Unexpected Exception, this is probably a bug: '/api/v3/plugin/ansible/content/published/collections/index/ansible/netcommon/versions/'
the full traceback was:

Traceback (most recent call last):
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/resolvelib/resolvers.py", line 171, in _merge_into_criterion
    crit = self.state.criteria[name]
KeyError: 'ansible.netcommon'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/thenmozhi.gopal/.local/bin/ansible-galaxy", line 135, in <module>
    exit_code = cli.run()
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible/cli/galaxy.py", line 552, in run
    return context.CLIARGS['func']()
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible/cli/galaxy.py", line 75, in method_wrapper
    return wrapped_method(*args, **kwargs)
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible/cli/galaxy.py", line 1188, in execute_install
    artifacts_manager=artifacts_manager,
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible/cli/galaxy.py", line 1217, in _execute_install_collection
    artifacts_manager=artifacts_manager,
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible/galaxy/collection/__init__.py", line 554, in install_collections
    upgrade=upgrade,
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible/galaxy/collection/__init__.py", line 1365, in _resolve_depenency_map
    max_rounds=2000000,  # NOTE: same constant pip uses
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/resolvelib/resolvers.py", line 453, in resolve
    state = resolution.resolve(requirements, max_rounds=max_rounds)
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/resolvelib/resolvers.py", line 347, in resolve
    failure_causes = self._attempt_to_pin_criterion(name, criterion)
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/resolvelib/resolvers.py", line 207, in _attempt_to_pin_criterion
    criteria = self._get_criteria_to_update(candidate)
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/resolvelib/resolvers.py", line 199, in _get_criteria_to_update
    name, crit = self._merge_into_criterion(r, parent=candidate)
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/resolvelib/resolvers.py", line 173, in _merge_into_criterion
    crit = Criterion.from_requirement(self._p, requirement, parent)
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/resolvelib/resolvers.py", line 79, in from_requirement
    cands = build_iter_view(provider.find_matches([requirement]))
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible/galaxy/dependency_resolution/providers.py", line 214, in find_matches
    coll_versions = self._api_proxy.get_collection_versions(first_req)
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible/galaxy/collection/galaxy_api_proxy.py", line 104, in get_collection_versions
    requirement,
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible/galaxy/collection/galaxy_api_proxy.py", line 102, in <genexpr>
    (version, api)
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible/galaxy/collection/galaxy_api_proxy.py", line 82, in _get_collection_versions
    raise last_error
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible/galaxy/collection/galaxy_api_proxy.py", line 61, in _get_collection_versions
    versions = api.get_collection_versions(requirement.namespace, requirement.name)
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible/galaxy/api.py", line 126, in wrapped
    return method(self, *args, **kwargs)
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible/galaxy/api.py", line 870, in get_collection_versions
    error_context_msg=error_context_msg, cache=True)
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible/module_utils/api.py", line 157, in run_function
    return call_retryable_function()
  File "/home/thenmozhi.gopal/.local/lib/python3.6/site-packages/ansible/galaxy/api.py", line 396, in _call_galaxy
    path_cache = self._cache[cache_id][url_info.path]
KeyError: '/api/v3/plugin/ansible/content/published/collections/index/ansible/netcommon/versions/'
Because of this error, i am not able to run regression test.  Other than this I have addressed all other review comments.

Reply:
this issue was resolved after adding --server="https://old-galaxy.ansible.com" --ignore-certs. Attached regression report with the -vvvv option.

Reply:
Thank you for adding the version information and posting the verbose regression test result summary.

I will mark this issue "Resolved".